pkg/server/server.go

```diff
@@ -192,8 +192,8 @@ func (s *ProxyServer) Proxy(stream client.ProxyService_ProxyServer) error {
 	if !ok {
 		return fmt.Errorf("failed to get context")
 	}
-	userAgent := md.Get(header.UserAgent)
-	klog.Infof("proxy request from client, userAgent %s", userAgent)
+	caller := md.Get("caller")
```

Comment:
Why is this change?

Reply:
GRPC User Agent is tied to a connection, and this was fine before because each proxy request involved creating a new connection. Now that we're using one connection, we need to differentiate each stream with a different caller.